### ISSUE TITLE
Consider adding license to your project ;)

### ISSUE THREAD

**Issue by bekicot** [everypolitician-scrapers/indonesia#1]
Hello,
License is important for others who want to use your repository. 
I've searched through your repository, but cant find anything mentioning license.

If you want to let people do anything they want with your code as long as they provide attribution back to you and don’t hold you liable. I recommend you to add MIT License.

**Comment by jayvdb** [comment]
@gombang, you will be interested in this project ;-)

**Comment by gombang** [comment]
Indeed. I think I have checked it out once, but forgot to follow up.
